proxy/src/util.rs

```diff
@@ -77,6 +80,21 @@ pub fn filter_capabilities_in_status(status: &mut Status) {
     }
 }
 
+// Remove unsupported capabilities in command continuation request response.
+pub fn filter_capabilities_in_continuation(continuation: &mut CommandContinuationRequest) {
+    if let CommandContinuationRequest::Basic(basic) = continuation {
+        if let Some(Code::Capability(capabilities)) = basic.code() {
+            let capabilities = filter_capabilities(capabilities.clone());
```

Comment:
Hmm, all these filter methods have a lot of unnecessary clones. Wouldn't be necessary if we refactor them from `fn foo(bar: &mut Bar)` to `fn foo(bar: Bar) -> Bar`. However, low priority.